src/components/contexts/error-handler.tsx

```diff
@@ -42,6 +45,7 @@ const ErrorHandler: React.FC = ({ children }) => {
       return;
     }
 
+    setErrorMessage(err.message);
     console.error(err);
```

Comment:
```suggestion
```
Maybe could be removed as well? Has the setErrorMessage above already.

Reply:
setErrorMessage is for client to see but console.error could be useful for testing etc. It has the whole error message in it.